### PR TITLE
Sketch out message type contents a bit more.

### PR DESCRIPTION
**Pull request by tomikazi**
Capturing our morning discussion in term of actual E2 proto changes.

### CODE REVIEW

File: api/sb/e2-interface.proto [hunk 1]
```diff
@@ -34,17 +32,73 @@ message TelemetryRequest {
 
 // TelemetryMessage carries sensory data about the RAN environment.
 message TelemetryMessage {
-    // one of the telemetry messages sent from base station components to the controller
+    E2MessageType messageType = 1;
+    oneof s {
+        E2RadioMeasReportPerUE radioMeasReportPerU = 2;
```
Comment (woojoong88) [inline api/sb/e2-interface.proto]: Sorry it was my fault. Could you add the character 'E" at the end of the variable name?

Comment (woojoong88) [reply api/sb/e2-interface.proto]: Otherwise, looks good to me.

Comment (shadansari) [reply api/sb/e2-interface.proto]: Do we want to drop the E2 prefix as well?

**Comment by tomikazi** [comment]
Yes, good idea. I can do that.

**Pull request merged**